docs/source/bond_family_guide/sysadmin_guide/installation.rst

```diff
@@ -0,0 +1,477 @@
+   otherwise. Any license under such intellectual property rights must be
+   express and approved by Intel(R) in writing.
+
+************
+Installation
+************
+
+Overview
+========
+
+This article is part of the Sawtooth Lake Bond Family documentation.
+
+Please view the :doc:`eula` before proceeding.
+
+This article guides you through the installation of the Sawtooth Lake Bond
+Family validator on a single server or device, along with required software
+tools and utilities. The following tasks are outlined:
+
+1. Enable SGX
+2. Install required tools
+
+  - Python, Python Extensions, and Python dependencies
+  - Intel Management Engine Components
+  - Intel Software Guard Extensions Platform
+
+3. Install Sawtooth Lake validator
+
+Hardware and Operating System Requirements
+==========================================
+
+- This document assumes that your node runs on provided Hewlett-Packard (HP)
+  hardware. If you are using different hardware, be advised that:
+
+  + The configuration steps to configure SGX may differ from those in this document
+  + The hardware must support `Intel SGX <https://software.intel.com/en-us/sgx>`_.
+
+- Supported OS versions: Windows 10 and Windows 7
+
+- If you need the network drivers for your HP workstation, they can downloaded
+  `here <http://ftp.hp.com/pub/softpaq/sp72501-73000/sp72518.exe>`_. 
+
+- Installation of the validator requires the use of a service account. The 
+  validator will not run under a user with admin privileges. 
+
+
+Networking
+==========
+
+Networking for your validator node must be set up before your node will be able
+to communicate with other nodes in the network:
+
+- Your computer or device must be connected to your LAN/WAN via the integrated 
+  network interface.
+- The node requires a publicly-addressable IP address or fully-qualified 
+  domain name (FQDN).
+- The node may be located behind a NAT firewall, may be given a fully exposed 
+  public network address, or may be on a DMZ.
+
+The following additional requirements apply:
+
+- An open TCP port on the publicly addressable IP address that maps to the 
+  node's configured TCP/http port.
+  
+  + The default TCP port is 8800, but your configuration may differ. 
+
+- An open UDP port on the publicly addressable IP address that maps to 
+  node's configured UDP/gossip layer. 
+  
+  + The default UDP port is 5500, but your configuration may differ. 
+
+Configuration of the validator node, including networking, is covered in
+:doc:`configuration`.
+
+For technical questions related to network, please see :doc:`contacts`. 
+
+
+Before You Begin
+================
+
+Download Required Software
+--------------------------
+
+A number of software packages and tools are required by Sawtooth Lake.
+Download the required software by clicking the links below.
+
+.. note:: The Sawtooth Lake validator software is provided to you by your 
+    Intel representative. Refer to :doc:`contacts`.
```

Comment:
@dcmiddle and @mfford - reaching out to you or someone to resolve the need for a contact to get stuff for the Bond network.